sdk/formrecognizer/ai-form-recognizer/src/constants.ts

```diff
@@ -1,11 +1,23 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+/**
+ * Defines known cloud audiences for Form Recognizer.
```

Comment:
Would be helpful to customers to have a brief description of what this means, practically speaking.